setup.py

```diff
@@ -1,33 +1,33 @@
 from setuptools import setup, find_packages
 
-with open("README.rst") as readme_file:
+with open('README.md') as readme_file:
     readme = readme_file.read()
 
-requirements = ["girder>=3.0.0a1"]
+requirements = ['girder>=3.0.0a1']
 
 setup(
-    author="Kitware, Inc.",
-    author_email="kitware@kitware.com",
+    author='Kitware, Inc.',
+    author_email='kitware@kitware.com',
     classifiers=[
-        "Development Status :: 2 - Pre-Alpha",
-        "License :: OSI Approved :: Apache Software License",
-        "Natural Language :: English",
-        "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
+        'Development Status :: 2 - Pre-Alpha',
+        'License :: OSI Approved :: Apache Software License',
+        'Natural Language :: English',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
     ],
-    description="A Girder plugin for data import tracking in HistomicsUI",
+    description='A Girder plugin for data import tracking in HistomicsUI',
     install_requires=requirements,
-    license="Apache Software License 2.0",
+    license='Apache Software License 2.0',
     long_description=readme,
-    long_description_content_type="text/x-rst",
+    long_description_content_type='text/x-rst',
```

Comment:
This should be changed to the markdown mimetype to match README.md.

Reply:
Good catch, thanks!